cli/cmd/root.go

```diff
@@ -215,6 +216,64 @@ func getPercentTLS(stats *pb.BasicStats) float64 {
 	return float64(stats.TlsRequestCount) / float64(reqTotal)
 }
 
+type cniPluginOptions struct {
```

Comment:
To help keep things organized, let's move this struct definition into the file where it's used -- `cli/cmd/install-cni-plugin.go`. You can also move the `newCNIPluginOptions`, `validate`, and `taggedCNIPluginImage` funcs as well. That's more consistent with how other CLI commands are setup (the `proxyConfigOptions` struct in this file is an exception to that rule, since it applies to more than one command).